draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -336,7 +336,7 @@ Implementations and operators of catalog zones may choose to provide their own p
 Custom properties can occur both globally, or for a specific member zone.
 To prevent a name clash with future properties, such properties should be represented below the label `ext`.
 
-`ext` is not a placeholder, so a custom property would have domains names as follows:
+`ext` is not a placeholder, so a custom property would have domain names as follows:
```

Comment:
I'm not sure 'domain' is appropriate here.

Reply:
@mind04 See the suggestion to resolve this below

Reply:
Resolved by the suggestion